CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ add_subdirectory(doc/examples/motion_planning_api)
 add_subdirectory(doc/examples/motion_planning_pipeline)
 add_subdirectory(doc/examples/move_group_interface)
 add_subdirectory(doc/examples/moveit_cpp)
+add_subdirectory(doc/examples/parallel_planning)
```

Comment:
The examples page has this note:
```
a catch-all place for pages that have not yet been updated for the new structure or are still not yet ported from ROS 1.

To migrate these pages into the new structure please see the appropriate pages under [Contributing](https://moveit.picknik.ai/main/doc/how_to_contribute/how_to_contribute.html).
```

You've made a nice ROS2 tutorial here, so I think it should be placed as according to the new structure! It sounds like this would fit right in with the other How-To guides.